### PR TITLE
WRO-697: Use babel-preset-enact's babel config

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
There are three repos (cli, storybook-utils, eslint-config-enact) that set the babel config.
It is necessary to change the structure so that this babel config is integrated into one preset and used in three repos.
This makes maintenance and management easier.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Use babel-preset-enact's babel config

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-697

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Updated `babel.config.js` to use `babel-preset-enact` that supporting own enact app config.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
### pack

* Updated `babel.config.js` to use `babel-preset-enact` module that holds the babel config for Enact.
```

**Review by MikyungKim — COMMENTED** [review]
Should we add `babel-preset-enact` to the list here https://github.com/enactjs/cli/blob/master/commands/info.js#L108 ?

**Comment by hong6316** [comment]
> Should we add `babel-preset-enact` to the list here https://github.com/enactjs/cli/blob/master/commands/info.js#L108 ?

I think yes, I will add a new line.

File: package.json [hunk 2]
```diff
@@ -43,29 +43,16 @@
...
-    "@babel/preset-typescript": "^7.18.6",
-    "@babel/runtime": "^7.20.7",
-    "@enact/dev-utils": "^5.1.0",
+    "@enact/dev-utils": "github:enactjs/dev-utils#feature/WRO-697",
     "@enact/template-sandstone": "^1.5.2",
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "@typescript-eslint/eslint-plugin": "^5.48.2",
```
Comment (MikyungKim) [inline package.json:62]: why this module has been removed?

File: package.json [hunk 3]
```diff
@@ -43,29 +43,16 @@
     "upupdowndownleftrightleftrightbastart"
   ],
   "dependencies": {
-    "@babel/core": "^7.20.12",
```
Comment (MikyungKim) [inline package.json:46]: I guess CLI has requirements for "@babel/core" in license and transpile js files.
Can we remove this?

File: package.json (hunk 2 above)
Comment (hong6316) [reply package.json:62]: I misunderstood unify scope. I will restore and will investigate this module in WRP-6904

File: package.json (hunk 3 above)
Comment (hong6316) [reply package.json:46]: Yes, the "babel-preset-enact" repo has "@babel/core", and "cli" include this preset(that module is still included in the "cli" repo).
`enact license/transpile` command also passed in cli automation tests

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**